### DISCUSSION TITLE
需要几张 🍄XSOSI 作报告的数据分析图

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#287] [2️⃣ 聊项目]
![image](https://github.com/X-lab2017/open-wonderland/assets/15010826/89c93467-81af-48c8-85bf-0441ee073309)

开放原子峰会，关于教育和人才评估方面的报告，需要几张分析图用来做案例，看看帮忙提供下~ @frank-zsy 

1. 教师 @will-ww 三个月在 oss101 上的增长变化
2. 助教 @TieWay59 @zhicheng-ning 三个月在 oss101 上的增长变化
3. 学员 DarkHighness、Liuyushiii 等三个月在 oss101 上的增长变化
4. oss101 仓库三个月的变化
5. 其他值得拿出来分享的内容均可

**Comment by frank-zsy** [comment]
![image](https://github.com/X-lab2017/open-wonderland/assets/8512426/1528aaa0-7e98-4205-a32e-c45a6cdc2983)

我把几个账号的数据放在一起了，但 TieWay 的可能有点太高，导致后面的比较压缩了。

另外，Hypercrx 的直接截图是挺好的

活跃度
![image](https://github.com/X-lab2017/open-wonderland/assets/8512426/e0031249-79a7-481a-9e5d-51761f050da0)

影响力 
![image](https://github.com/X-lab2017/open-wonderland/assets/8512426/e1f35c86-eec8-4e6d-a07a-e20537747e93)

参与人数
![image](https://github.com/X-lab2017/open-wonderland/assets/8512426/9cd4cb8b-1a55-49b5-ab12-e2b7a9d3b40d)

Issue 情况
![image](https://github.com/X-lab2017/open-wonderland/assets/8512426/c70bb8ed-dccc-482d-b581-685186821d04)

PR 情况
![image](https://github.com/X-lab2017/open-wonderland/assets/8512426/60eab285-a0e5-4f14-a838-db42fe540411)

**Comment by will-ww** [comment]
可以做几张能否反映 XSOSI 一年整体情况的趋势图和状态图吗？@frank-zsy

我能想到的几个包括：
- 基于 Codepen 的组织层面（多项目）的价值网络图
- 组织层面的 仓库数、Star 数、参与人数、Activity、OpenRank 的变化趋势图
- 项目的平均 OpenRank 变化趋势、平均 Activity 变化趋势、个人在 org 上的 OpenRank 变化趋势

可以看看哪些好拉出来，先拉点出来即可，不好做的晚点再说~

**Reply by frank-zsy** [reply]
所有的数据都是公开可用的，王老师这个数据是要在 WAIC 上用吗？那是否要等到六月的数据也结算以后再统一出？

**Reply by will-ww** [reply]
是的，除了 WAIC，陆续会有不少对外的活动，可以 6 月份统一出~

**Reply by will-ww** [reply]
同时，咱们也加入：https://github.com/X-lab2017/gi-assets-xlab 项目吧~ @frank-zsy

**Reply by frank-zsy** [reply]
好的，这个月会加上

**Comment by zhicheng-ning** [comment]
> 我能想到的几个包括：
> 
> * 基于 Codepen 的组织层面（多项目）的价值网络图
> * 组织层面的 仓库数、Star 数、参与人数、Activity、OpenRank 的变化趋势图
> * 项目的平均 OpenRank 变化趋势、平均 Activity 变化趋势、个人在 org 上的 OpenRank 变化趋势

王老师，这一块儿的变化趋势图是直接交付几张单独的图表，还是交付一个大屏，里面包含这些图表呢？

**Reply by will-ww** [reply]
不用大屏，先用一个 notebook 做几个即可~

**Comment by zhicheng-ning** [comment]
持续更新：https://dataease.x-lab.info/link/AwTy29Zk
- [ ] 基于 Codepen 的组织层面（多项目）的价值网络图
- [x] 仓库数变化趋势图
- [x] 总津贴变化趋势图
- [ ] Star 数变化趋势图
- [x] 参与人数变化趋势图
- [ ] Activity变化趋势图
- [x] OpenRank变化趋势图
- [x] 项目的平均 OpenRank 变化趋势
- [x] 个人在 org 上的 OpenRank 变化趋势

**Reply by will-ww** [reply]
So cool~

**Reply by zhicheng-ning** [reply]
<img width="1920" alt="image" src="https://github.com/X-lab2017/open-wonderland/assets/39022409/288d45e8-1b01-47eb-80ad-50d4da3d3cfd">

**Comment by frank-zsy** [comment]
在 Notebook 中实现了 [XSOSI 年度报告](https://github.com/X-lab2017/open-digger/blob/master/notebook/community_analysis/xsosi-annual-report.ipynb)，包含如下内容：

- [ ] 基于 Codepen 的组织层面（多项目）的价值网络图
- [x] 仓库数变化趋势图
- [x] Star 数变化趋势图
- [x] 参与人数变化趋势图
- [x] Activity变化趋势图
- [x] OpenRank变化趋势图
- [x] 项目的平均 OpenRank 变化趋势
- [x] 个人在 org 上的 OpenRank 变化趋势

由于 Notebook 中目前无法渲染网络图，多项目价值网络图会在 Codepen 上单独实现。

**Comment by zhicheng-ning** [comment]
> * [x]  参与人数变化趋势图

- 参与人数是不仅限于每期参加 XSOSI 的人数吗

- 参与项目的数量可能有点问题

<img width="844" alt="image" src="https://github.com/X-lab2017/open-wonderland/assets/39022409/d8b9003d-71a0-4cb5-8447-ff7b266ea7b5">
